examples/arm/executor_runner/arm_executor_runner.cpp

```diff
@@ -349,7 +351,10 @@ int main(int argc, const char* argv[]) {
   ET_LOG(Info, "Input prepared.");
 
   ET_LOG(Info, "Starting the model execution...");
+  StartMeasurements();
```

Comment:
Did you look at the ET devtools APIs around this for ETDump?
https://pytorch.org/executorch/stable/sdk-etdump.html

Reply:
Yes I really like it and I think this might be a more proper way to collect runtime stats going forward, just need to figure out how to dump that file out from our embedded systems in a good way :) We can probably use it quite easy in the FVP setup when we run in semi hosting mode that have access to the file system.

Reply:
Sounds good. Thanks.